deepchem/hyper/pyGPGO_classes.py

```diff
@@ -0,0 +1,855 @@
+import numpy as np
```

Comment:
This is a lot of code now that I'm looking at it. We should discuss whether we want to adapt this much code into DeepChem to remove the pyGPGO dependency.

@brianpzaide Can you come by the DeepChem OH sometime this week (every weekday at 10:30 pm IST)?

Reply:
@rbharath Sure, I will be there